modular_citadel/code/modules/mob/living/simple_animal/simplemob_vore_values.dm

```diff
@@ -133,3 +133,51 @@
 	vore_active = TRUE
 	isPredator = TRUE
 	vore_default_mode = DM_DIGEST
+
+/mob/living/simple_animal/butterfly/holo
```

Comment:
I strongly suggest you to modify how /obj/effect/holodeck_effect/mobspawner/activate() works instead of making more snowflakes holomob subtypes, it's a more permanent solution and also includes bees, holocarps, holomonkeys and penguins.
Basically, do this instead:
```

/obj/effect/holodeck_effect/mobspawner/activate(var/obj/machinery/computer/holodeck/HC)
	if(islist(mobtype))
		mobtype = pick(mobtype)
	mob = new mobtype(loc)

	// these vars are not really standardized but all would theoretically create stuff on death
	for(var/v in list("butcher_results","corpse","weapon1","weapon2","blood_volume") & mob.vars)
		mob.vars[v] = null
	ENABLE_BITFIELD(mob.flags_1, HOLOGRAM_1)
	if(isliving(mob))
		var/mob/living/L = mob
		L.feeding = FALSE
		L.devourable = FALSE
		L.digestable = FALSE
	return mob
```